docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.9"
+
+services:
+  playground:
+    command: bash -c "cd /usr/src/app && v run server.v"
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "5555:5555"
+    volumes:
+      - .:/usr/src/app
```

Comment:
I am just curious, is there a convention or requirement for using `/usr/src/app`?

Reply:
Yeah, there is a convention when creating containers for apps to use that dir, not sure why 😅

Reply:
thanks